### PR TITLE
MWPW-133598 Workfront Login

### PR DESCRIPTION
**Pull request by meganthecoder**
* Add new Workfront Login block

Resolves: [MWPW-133598](https://jira.corp.adobe.com/browse/MWPW-133598)

**Test URLs:**
- Before: https://main--bacom--adobecom.hlx.live/drafts/methomas/workfront-login?martech=off
- After: https://methomas-workfront-login--bacom--adobecom.hlx.live/drafts/methomas/workfront-login?martech=off

### CODE REVIEW

File: blocks/workfront-login/workfront-login.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  background-position: center right calc(35px / 4);
+  background-size: calc(35px / 2) calc(35px / 2);
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='%23eb1000' viewBox='-2 -2 7 7'%3e%3cpath stroke='%23d9534f' d='M0 0l3 3m0-3L0 3'/%3e%3ccircle r='.5'/%3e%3ccircle cx='3' r='.5'/%3e%3ccircle cy='3' r='.5'/%3e%3ccircle cx='3' cy='3' r='.5'/%3e%3c/svg%3E");
+}
+
+.workfront-login .input-group-append {
+  color: #505050;
+  font-size: 14px;
+  line-height: 1.5;
+  margin-left: 8px;
```
Comment (overmyheadandbody) [inline blocks/workfront-login/workfront-login.css:67]: Is there a need to reflect this for RTL? Same question for L31, L56, L84, L110

**Review by overmyheadandbody — APPROVED** [review]

File: blocks/workfront-login/workfront-login.css (hunk 1 above)
Comment (meganthecoder) [reply blocks/workfront-login/workfront-login.css:82]: Updated

RTL example: https://methomas-workfront-login--bacom--adobecom.hlx.live/ae_ar/drafts/methomas/workfront-login

File: blocks/workfront-login/workfront-login.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    try {
+      const resp = await fetch(checkURL, { signal: controller.signal });
+      if (resp.ok) {
+        location.redirect(checkURL);
+      } else {
+        showError();
+      }
+    } catch (err) {
+      if (!err.code) {
+        location.redirect(checkURL);
```
Comment (Brandon32) [inline blocks/workfront-login/workfront-login.js:93]: Looks like if there's no response we redirect anyways? What is `err.code`?

**Review by Brandon32 — APPROVED** [review]

**Review by JasonHowellSlavin — APPROVED** [review]
Great work!

File: blocks/workfront-login/workfront-login.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.workfront-login {
+  --error: #c9252d;
+
```
[the remark this answers is not in the document]
Comment (meganthecoder) [reply blocks/workfront-login/workfront-login.css:13]: Stylelint told me to

File: blocks/workfront-login/workfront-login.js (hunk 2 above)
Comment (Brandon32) [reply blocks/workfront-login/workfront-login.js:93]: If you type in an incorrect subdomain for the .my.workfront.com input field it checks if that is a valid URL and redirects anyways. Try test and you will get `test.my.workfront.com’s server IP address could not be found.`.

**Pull request merged**